docs/Apps/Metabolomic Data/Dual Mode Visualization.md

```diff
@@ -273,53 +274,111 @@ Once the parameters are specified, click on the *Update* button to plot the volc
 
 *Filtered Metabolites Visualization* provides the visualization of cohort-based distribution of the metabolites that are significant based on the parameters specified.
 
-![Filtered Metabolites Visualization](../../img/DualMode/metab_app_filtered_metab_viz.png) <center>**Figure 41.** Filtered Metabolites Visualization</center>
+![Filtered Metabolites Visualization](../../img/DualMode/metab_app_stats_analysis_filtered_metabolites_visualization.png) <center>**Figure 41.** Filtered Metabolites Visualization</center>
 
 *Filtered Normalized Table* contains the normalized data of the metabolites that are significant based on the parameters specified.
 
-![Filtered Normalized Table](../../img/DualMode/metab_app_filtered_normalized_table.png) <center>**Figure 42.** Filtered Normalized Table</center>
+![Filtered Normalized Table](../../img/DualMode/metab_app_stats_analysis_filtered_normalised_table.png) <center>**Figure 42.** Filtered Normalized Table</center>
 
 *Filtered Differential Expression Table* contains only the metabolites that have significant *p*-values as specified.
 
-![Filtered Differential Expression Table](../../img/DualMode/metab_app_filtered_diff_exp.png) <center>**Figure 43.** Filtered Differential Expression Table</center>
+![Filtered Differential Expression Table](../../img/DualMode/metab_app_stats_analysis_filtered_diff_exp_table.png) <center>**Figure 43.** Filtered Differential Expression Table</center>
 
 *Differential Expression Table* contains all the differentially expressed metabolites without any filtering.
 
-![Differential Expression Table](../../img/DualMode/metab_app_nonfiltered_diff_exp.png) <center>**Figure 44.** Differential Expression Table</center>
+![Differential Expression Table](../../img/DualMode/metab_app_stats_analysis_diff_exp_table.png) <center>**Figure 44.** Differential Expression Table</center>
+
+*Pathway Enrichment Analysis* performs the pathway enrichment analysis for the significant metabolites based on the parameters specified for the particular cohort comparison. Click on the *Perform Pathway Analysis* button.
+As a result, you get Metabolite Set Enrichment Analysis and Pathway Topology Analysis plots that can be downloaded under the *Plot* panel. You can also obtain the tablular representation of the plots by selecting onto the *Table* panel.
```

Comment:
You can click on the _Perform Pathway Analysis_ button to get metabolite set enrichment analysis and....